examples/installation/single_account_deployment/variables.tf

```diff
@@ -74,8 +86,14 @@ variable "gw_count" {
 variable "web_console_admin_password" {
   sensitive   = true
   type        = string
-  default     = null # Random
-  description = "Admin password (Random generated if not set)"
+  default     = null
+  description = "Admin user password. If not set, and web_console_admin_password_secret_name is also not set, random value is generated."
```

Comment:
Sorry I know it's my phrasing, but 'a' is missing before random: "a random value is generated"